tooling/Microsoft.VisualStudio.LanguageServices.Blazor/BlazorProjectEngineFactory.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Blazor
 {
-    [ExportCustomProjectEngineFactory("Blazor-0.1", SupportsSerialization = false)]
+    [ExportCustomProjectEngineFactory("Blazor-0.2", SupportsSerialization = false)]
```

Comment:
TBH I'm unclear why it's necessary to specify a version in places like this. It looks like an opaque string. If there was no version number to change here, wouldn't people just get whatever version they have installed? How does having a version number here help us?

Reply:
You're right, it is an opaque string. This string matches what we're doing in MSBuild.

The value of `DefaultRazorConfiguration` from MSBuild flows through to the razor tools, and that filters `ICustomProjectEngineFactor` implementations based on these attributes.

Reply:
I've added comments to these places explaining that it's a wire-up from MSBuild.